aperturedb/Operations.py

```diff
@@ -0,0 +1,28 @@
+class Operations(object):
```

Comment:
for future reference, we support more operations and parameters: https://docs.aperturedata.io/parameters/operations.html

Reply:
https://github.com/aperture-data/aperturedb-python/issues/160